flake.nix

```
@@ -3,13 +3,22 @@
 
   inputs = {
     nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+
+    hjem = {
+      url = "github:feel-co/hjem";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+    ndg = {
+      url = "github:feel-co/ndg/v2";
+    };
   };
 
   outputs = {
     self,
     nixpkgs,
+    ndg,
     ...
-  }: let
+  } @ inputs: let
```

`@ inputs` isn't necessary if we're not using it. Especially in this flake since we've got like 3 inputs, one of them being nixpkgs.

```suggestion
  }: let
```

---

We are using it (it's passed into `docs/package.nix` as a special argument)

---

In that case, just use ndg as inputs.ndg rather than passing it through the attrset.